### PR TITLE
CIRC-813 fix issue with URI Too Large error

### PR DESCRIPTION
**Pull request by roman-barannyk**
## Approach 
- Use FindWithMultipleCqlIndexValues to fetch open loans for items to avoid 414 Uri too long error;
- Cover this scenario by test;

Resolves: [CIRC-813](https://issues.folio.org/browse/CIRC-813)

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [x] There are no breaking changes in this PR.

### CODE REVIEW

File: src/test/java/api/requests/ItemsInTransitReportTests.java [hunk 1]
```diff
@@ -592,4 +623,14 @@ private ItemBuilder createSmallAngryPlanetItemBuilder() {
       .withVolume("smallAngryPlanetVolume")
       .withYearCaption(Collections.singletonList("2019"));
   }
+
+  private ItemBuilder createSmallAngryPlanetItemBuilder(UUID locationId, int barcode) {
```
Comment (alexanderkurash) [inline src/test/java/api/requests/ItemsInTransitReportTests.java:627]: Do we need this to be a separate method, considering it's only called from one place?

File: src/test/java/api/requests/ItemsInTransitReportTests.java [hunk 2]
```diff
@@ -592,4 +623,14 @@ private ItemBuilder createSmallAngryPlanetItemBuilder() {
...
+
+    return new ItemBuilder()
+      .withPermanentLoanType(materialTypesFixture.book().getId())
+      .withMaterialType(loanTypesFixture.canCirculate().getId())
+      .withBarcode(Integer.toString(barcode))
```
Comment (alexanderkurash) [inline src/test/java/api/requests/ItemsInTransitReportTests.java:632]: Barcode is a string, so why do we pass it as an integer just to convert to a string at the end?

File: src/test/java/api/requests/ItemsInTransitReportTests.java [hunk 3]
```diff
@@ -582,6 +604,15 @@ private InventoryItemResource createSmallAngryPlanet() {
       itemsFixture.thirdFloorHoldings());
   }
 
+  private InventoryItemResource createSmallAngryPlanet(UUID locationId, int barcode) {
```
Comment (alexanderkurash) [inline src/test/java/api/requests/ItemsInTransitReportTests.java:607]: I think `createSmallAngryPlanetCopy` would probably describe what this method does better.

File: src/test/java/api/requests/ItemsInTransitReportTests.java [hunk 4]
```diff
@@ -443,6 +443,28 @@ public void reportItemsInTransitSortedByCheckInServicePoint() {
...
   }
 
+  @Test
+  public void reportWillNotFailWithUriTooLargeError() {
+
+    final UUID firstServicePointId = servicePointsFixture.cd1().getId();
+    final UUID forthServicePointLocationId = locationsFixture.fourthServicePoint().getId();
+
+    for (int i = 0; i < 200; i++) {
+      InventoryItemResource item = createSmallAngryPlanet(forthServicePointLocationId, i);
```
Comment (alexanderkurash) [inline src/test/java/api/requests/ItemsInTransitReportTests.java:453]: Convert barcode to string here?

File: src/test/java/api/requests/ItemsInTransitReportTests.java (hunk 1 above)
Comment (roman-barannyk) [reply src/test/java/api/requests/ItemsInTransitReportTests.java]: moved

File: src/test/java/api/requests/ItemsInTransitReportTests.java (hunk 2 above)
Comment (roman-barannyk) [reply src/test/java/api/requests/ItemsInTransitReportTests.java]: done

File: src/test/java/api/requests/ItemsInTransitReportTests.java (hunk 3 above)
Comment (roman-barannyk) [reply src/test/java/api/requests/ItemsInTransitReportTests.java]: renamed

File: src/test/java/api/requests/ItemsInTransitReportTests.java (hunk 4 above)
Comment (roman-barannyk) [reply src/test/java/api/requests/ItemsInTransitReportTests.java]: done

**Pull request merged**